pkgs/applications/misc/keepass-plugins/keeotp2/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, buildEnv, fetchurl, mono }:
+
+let
+  version = "1.5.9";
+  drv = stdenv.mkDerivation {
```

Comment:
Please use a recursive pattern here, instead of inheriting the version. For further information, [see here](https://github.com/tiuub/KeeOtp2/releases/download/v${version}/KeeOtp2.plgx).